### PR TITLE
[tsdoc-cleanup] clean up tsdoc docblocks

### PR DESCRIPTION
**Pull request by crookse**
No issue for this. I was converting Sockets to use TSDoc and decided to check Drash's doc block formatting and the `@return` vs `@returns` annotations.

### CODE REVIEW

File: src/http/response.ts [hunk 1]
```diff
@@ -217,10 +216,9 @@ export class Response {
   /**
    * Get the status message based on the status code.
    *
-   * @return
-   *     Returns the status message associated with this.status_code. For
-   *     example, if the response's status_code is 200, then this method
-   *     will return "OK" as the status message.
+   * @return The status message associated with this.status_code. For example,
```
Comment (ebebbington) [inline src/http/response.ts:219]: `returns`

File: src/http/request.ts [hunk 2]
```diff
@@ -54,7 +54,7 @@ export class Request extends ServerRequest {
     this.original_request = originalRequest;
     this.url = originalRequest.url;
     this.url_path = this.getUrlPath(originalRequest);
-    this.url_query_params = this.getUrlQueryParams(originalRequest);
+    this.url_query_params = this.getUrlQueryParams();
```
[the remark this answers is not in the document]
Comment (ebebbington) [reply src/http/request.ts:57]: Nvm, saw that it was removed from the method as an arg

[the remark this answers is not in the document]
Comment (crookse) [reply src/http/request.ts:57]: Yea the it took an arg but it wasn’t being used and didn’t need to be used

File: src/http/response.ts (hunk 1 above)
Comment (crookse) [reply src/http/response.ts]: done: f729185

**Pull request merged**